app/javascript/src/Users/components/FeaturesFieldset.jsx

```diff
@@ -0,0 +1,107 @@
+// @flow
+
+import React from 'react'
+
+import { getFeatureName, canFeatureSetServicePermissions } from 'Users/utils'
+
+import type { Feature, AdminSections } from 'Users/types'
+
+type Props = {
+  features: Feature[],
+  selectedSections?: AdminSections[],
+  areServicesVisible?: boolean,
+  onAdminSectionSelected: AdminSections => void
+}
+
+/**
+ * Contains a set of Features that can be checked by the user.
+ * @param {Feature[]} features - An array containing all available Features in the fieldset.
+ * @param {AdminSections[]} selectedSections - An array containing all checked elements. Besides Features it can be the checkbox for 'all services enabled'.
+ * @param {boolean} areServicesVisible - Whether or not the fieldset with all services is visible.
+ * @param {function} onAdminSectionSelected - A callback function triggered when a feature is checked or unchecked.
+ */
+const FeaturesFieldset = ({ features, selectedSections = [], areServicesVisible = false, onAdminSectionSelected }: Props) => {
+  const featuresList = `FeatureAccessList ${areServicesVisible ? '' : 'FeatureAccessList--noServicePermissionsGranted'}`
+  const allServicesChecked = !selectedSections.includes('services')
+
+  return (
+    <fieldset>
+      <legend className='label'>This user can access</legend>
+      <ol className={featuresList}>
+        <input type='hidden' name='user[member_permission_ids][]' />
+        {features.map(feature =>
+          <FeatureCheckbox key={feature} value={feature} checked={selectedSections.includes(feature)} onChange={onAdminSectionSelected} />
+        )}
+        {areServicesVisible &&
+          <AllServicesCheckbox checked={allServicesChecked} onChange={onAdminSectionSelected} />
+        }
+      </ol>
+    </fieldset>
+  )
+}
+
+/**
+ * A checkbox representing a Feature the user will have access to.
+ * @param {Feature} value - The feature this checkbox represents.
+ * @param {boolean} checked - Whether or not this checkbox is selected.
+ * @param {Function} onChange - A callback function triggered when its value changes.
+ */
+const FeatureCheckbox = ({ value, checked, onChange }: {
+  value: Feature,
+  checked?: boolean,
+  onChange: Feature => void
+}) => {
+  const featuresListItem = `FeatureAccessList-item FeatureAccessList-item--${value} ${checked ? 'is-checked' : 'is-unchecked'}`
```

Comment:
Here I'd make it obvious that it's a list of classNames. This is related to https://github.com/3scale/porta/pull/604/files#r272902142